falco.yaml

```diff
@@ -177,6 +177,39 @@ rules_files:
   - /etc/falco/falco_rules.local.yaml
   - /etc/falco/rules.d
 
+# [Experimental] `rules`
```

Comment:
```suggestion
# [Incubating] `rules`
```

As per the upcoming [adoption policy](https://github.com/falcosecurity/falco/blob/master/proposals/20231220-features-adoption-and-deprecation.md#adoption-policy), this should be labeled as "Incubating".

This is not a blocker for this PR, but maybe it is worth starting using new labels.

Reply:
Agree